### PR TITLE
fix: Add new table `component_analysyes_requests`.

### PR DESCRIPTION
**Pull request by rafiu007**
Added new table for storing data from CA calls.Look at the doc below to see specs.
https://docs.google.com/document/d/1JxEYgMSVHPBevHvjdvPX_cRUeUO0Be-0nGEnOXvRNXc/edit?usp=sharing

### CODE REVIEW

File: f8a_worker/models.py [hunk 1]
```diff
@@ -428,3 +428,18 @@ class OSIORegisteredRepos(Base):
...
     last_scanned_at = Column(DateTime)
+
+
+class ComponentAnalysesRequests(Base):
+    """Table for CA request."""
+
+    __tablename__ = "component_analyses_requests"
+
+    component_analyses_request_id = Column(String(64), primary_key=True)
+    request_id = Column(String(64), nullable=True)
```
Comment (prashbnair) [inline f8a_worker/models.py:439]: This is should be a UUID. It will be generated for each request.

File: f8a_worker/models.py [hunk 2]
```diff
@@ -428,3 +428,18 @@ class OSIORegisteredRepos(Base):
...
     email_ids = Column(String(255), nullable=False)
     last_scanned_at = Column(DateTime)
+
+
+class ComponentAnalysesRequests(Base):
+    """Table for CA request."""
+
+    __tablename__ = "component_analyses_requests"
+
+    component_analyses_request_id = Column(String(64), primary_key=True)
```
Comment (prashbnair) [inline f8a_worker/models.py:438]: Should the generated primary key be a string or a number?

**Review by prashbnair — CHANGES_REQUESTED** [review]

File: f8a_worker/models.py (hunk 2 above)
Comment (rafiu007) [reply f8a_worker/models.py:438]: @prashbnair  Keeping it  as a string seems more flexible.the same has been done for other tables as well.

Comment (rafiu007) [reply f8a_worker/models.py]: Done.

File: f8a_worker/models.py (hunk 1 above)
Comment (rafiu007) [reply f8a_worker/models.py]: Done

**Review by prashbnair — APPROVED** [review]

**Pull request merged**